package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rdf-js",
+  "version": "4.0.1",
```

Comment:
I assume to increate version 4 which is latest `@types/rdf-js`.

Could also go with `2.0` which would be a major bump of the old [rdf-js](https://npm.im/rdf-js)

Reply:
hmm... this one is tricky. I would actually think it best to **reflect the major and minor semver of the spec**, but i'd like to hear what others have to say. Patch semantics would still let us make changes to the typings without features or breaking changes spec-wise.

Reply:
While I agree that aligning versioning with the spec, this might not always be possible.
Most of the times that the DT typings gained a major version increase, was due to changes in the typings that were not caused by spec changes.

For example: https://github.com/rdfjs/types/pull/2/commits/e6d3b8601a63e0a7e78c59c91a94d44ba325563b

Reply:
@blake-regalia we don't do SerVer of the spec right now so that would be little blocker

I also have the feeling that the types might introduce breaking changes not related to spec changes although I think we can strive to avoid that.

Which does not mean that `v4` is the one to go. I think I would propose to go "back" to `v2` which would be a major bump of the `rdf-js` package

Reply:
Ok I agree. Another reason could be breaking change for upgrading minimum TS version supported to use new language feature or something. Starting at `v2` sounds good to me @tpluscode

Reply:
Ok, it's 2.0 now